app/models/case/sar/offender.rb

```diff
@@ -73,6 +73,12 @@ def searchable_fields_and_ranks
   validate :validate_date_of_birth
   validate :validate_received_date
 
+  before_save :set_subject
```

Comment:
This will be triggered fro case create/update - is that intended or just for case creation?

Reply:
Intended to do it for update as well as create - so when someone edits the user's name, the subject changes too